imu_filter_madgwick/src/imu_filter_ros.cpp

```diff
@@ -517,3 +539,29 @@ void ImuFilterRos::checkTimeJump()
 
     reset();
 }
+
+bool ImuFilterRos::checkLargeTimeJumpForward(const ros::Time& imu_stamp)
+{
+    bool largeJump = false;
+    if (constant_dt_ > 0.0)
+    {
+        return largeJump;  // use constant dt
```

Comment:
This can be simplified to `return false;`.

Also, the comment is not very helpful. It should be something like "don't reject measurement on large jump if using constant dt". But why does that make sense? Shouldn't we do the opposite (always reject if using constant_dt)?

Reply:
We will change the comment and the return statement. If dt is constant, it doesn't matter what the jump in the timestamp is, so we should use the measurement.